sync3/conn.go

```diff
@@ -83,6 +84,7 @@ func (c *Conn) tryRequest(ctx context.Context, req *Request) (res *Response, err
 		if panicErr != nil {
 			err = fmt.Errorf("panic: %s", panicErr)
 			logger.Error().Msg(string(debug.Stack()))
+			sentry.GetHubFromContext(ctx).RecoverWithContext(ctx, panicErr)
```

Comment:
There won't ever be a hub though? I guess we do need the sentry middleware if we want to log contextual information about the request. Bleh.